build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     ext.coroutinesVersion = '1.3.9'
     ext.kotlin_ktx_version = '1.2.0'
     ext.wordPressUtilsVersion = 'develop-f98c841225fbc61031f32426aaef4e347b34f717'
-    ext.wordPressLoginVersion = 'develop-5007ecf737918be639e1d80ff2c6e07fd73f64cf'
+    ext.wordPressLoginVersion = '0.0.3'
```

Comment:
Had to release a new WPLoginAndroid version. I didn't notice at first that it required a new tag _before_ starting code freeze (like the scenario suggests to check) because when I checked the version before running the `code_freeze` lane, I failed to notice I was still on the `release/17.8` branch (where it was `0.0.2`) and not on `develop` (where it was a `develop-<sha>`). So I started the freeze, then got the warning that the strings from login lib wouldn't get translated, and aborted to do a 0.0.3 release, then restarted the lane at the point where it got interrupted (see 3731bb8)

This is why this change wasn't made on `develop` **prior** to the code freeze (as the scenario suggests) but instead was done **after** the branch was cut and is thus visible in this PR.